docs/docker-testnet.md

```diff
@@ -0,0 +1,65 @@
+# Join the testnet with Docker
+
+You must have [docker installed](https://docs.docker.com/engine/installation/) on a host machine.
+
+## Docker Image
+
+Docker image is automatically build and published on [DockerHub](https://hub.docker.com/r/aetrnty/epoch/). All tagged source code versions have their own docker image tag as well. Latest tagged ("stable") version is tagged with `latest`.
+Master branch of the source code is tagged as `dev`.
+
+To pull the latest "stable" image run:
+```bash
+docker pull aetrnty/epoch
+```
+
+## Start a Node
+
+To start a docker node and join the testnet run:
+```bash
+docker run -d --name epoch_node0 -p 3013:3013 aetrnty/epoch
+```
+
+Verify the node is running:
+```bash
+curl localhost:3013/v1/top
+```
+
+## Stop a Node
+
+To stop a docker node run:
+```bash
+docker stop epoch_node0
+```
+
+## Configuration
```

Comment:
How does user consult logs? (epoch*.log)

Reply:
They could like on any docker container `docker exec epoch_node0 cat log/epoch.log`

Reply:
A note where the logs are stored in the container would help the user I believe (the `docker exec epoch_node0 cat log/epoch.log` would be enough). Other option would be encouraging user to use volume for logs.

Reply:
updated in c8dcad2